activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/main/java/org/activiti/cloud/starter/rb/configuration/RuntimeBundleSwaggerConfig.java

```diff
@@ -52,9 +55,43 @@ public GroupedOpenApi runtimeBundleApi(@Value("${activiti.cloud.swagger.rb-base-
             .addOpenApiCustomizer(openApi ->
                 openApi.addExtension(BaseOpenApiBuilder.SERVICE_URL_PREFIX, swaggerBasePath)
             )
+            .addOpenApiCustomizer(openApiCustomizer())
             .build();
     }
 
+    public OpenApiCustomizer openApiCustomizer() {
+        return openAPI -> {
+            openAPI
+                .getPaths()
+                .values()
+                .forEach(val -> {
+                    val
+                        .readOperations()
+                        .forEach(operation -> {
+                            operation
+                                .getResponses()
+                                .forEach((key, value) -> {
+                                    if (key.matches("200")) {
+                                        Content contents = value.getContent();
+                                        String applicationHal = "application/hal+json";
+                                        String applicationJson = "application/json";
+                                        if (
+                                            contents != null &&
+                                            contents.containsKey(applicationHal) &&
+                                            contents.containsKey(applicationJson)
+                                        ) {
+                                            MediaType applicationHalValue = contents.remove(applicationHal);
+                                            MediaType applicationJsonValue = contents.remove(applicationJson);
+                                            contents.put(applicationJson, applicationJsonValue);
+                                            contents.put(applicationHal, applicationHalValue);
```

Comment:
Isn't the put enough to overwrite the data?

Reply:
No actually I want to change the sequence of the data so I have to remove it first otherwise applicationHal will always come before applicationJson. One thing I can do is to just remove applicationHal and add it again, it will then add after applicationJson

Reply:
https://github.com/search?q=org%3AActiviti%20hal%2Bjson&type=code is there a way to modify it like adding multiple options header in the right order in those call? maybe this will save us from this override?

Reply:
I tried changing the sequence of produces as seen in this commit -> [[`7421f48` (#1306)](https://github.com/Activiti/activiti-cloud/pull/1306/commits/7421f48f1b47be9efc32293a4e6ccb363cd20ef7)](url) but the acceptance tests were failing because all those is written for application/hal+json format. So response json is completely different so Elias suggested to go with this approach.